Core/Src/main.c

```diff
@@ -572,6 +624,25 @@ void move_ram_to_sd(void const * argument)
   /* USER CODE END move_ram_to_sd */
 }
 
+/* USER CODE BEGIN Header_transmit_ram */
+/**
+* @brief Function implementing the transmit_ram_ thread.
+* @param argument: Not used
+* @retval None
+*/
+/* USER CODE END Header_transmit_ram */
+void transmit_ram(void const * argument)
+{
+  /* USER CODE BEGIN transmit_ram */
+  /* Infinite loop */
+  for(;;)
+  {
+	  osDelay(2000);
```

Comment:
In the docs in dlm-high_level_functions, you say this should run every second. Not a big deal but make sure it matches up when you finalize the timing